pkgs/development/compilers/swift/compiler/default.nix

```diff
@@ -169,6 +170,21 @@ let
     chmod a+x "$targetFile"
   '';
 
+  # On Darwin, we need to use BOOTSTRAPPING-WITH-HOSTLIBS because of ABI
+  # stability, and have to provide the definitions for the system stdlib.
+  appleSwiftCore = stdenv.mkDerivation {
+    name = "apple-swift-core";
+    dontUnpack = true;
+
+    installPhase = ''
+      mkdir -p $out/lib/swift
+      cp -r \
```

Comment:
symlinking does not work?

Reply:
This is modelled after libs in `apple_sdk.nix`. I think the idea is that we try to avoid users having to download the full SDK.